tests/reproducibility/test_graphsage.py

```diff
@@ -169,3 +230,21 @@ def test_nai(petersen_graph, shuffle):
             petersen_graph, targets, [2, 2], tf.optimizers.Adam(1e-3), shuffle=shuffle
         )
     )
+
+
+# FIXME (#970): This test fails intermittently with shuffle=True
+@pytest.mark.parametrize("shuffle", [False])
+def test_link_prediction(petersen_graph, shuffle):
+    num_examples = 4
```

Comment:
Hm, `num_examples = 4` here with `batch_size = 4` means that there'll only ever be one batch? And thus this test isn't actually testing reproducibility with respect to batch orders?

Reply:
Good catch, I'll increase `num_examples` to 10